packages/gatsby-theme-carbon/src/components/GlobalSearch/Menu.js

```diff
@@ -69,10 +87,11 @@ const MenuItem = ({ page, index, onKeyDown, id }) => {
         className={className}
         to={page.path}
       >
-        {`${page.title} – `}
-        <span className={description}>
-          {page.description && page.description.toLowerCase()}
-        </span>
+        {page.title}&nbsp;
```

Comment:
What is the `&nbsp` doing? Nothing wrong with it! I just have no idea what that is

Reply:
It stands for non-breaking-space, it's just an alternative to the {" "} weirdness of JSX for introducing spaces.

Reply:
ah okay, thank you!